### ISSUE TITLE
Using - for output in unpack conflicts with progress output

### ISSUE THREAD

**Issue by ypujante** [nelenkov/android-backup-extractor#62]
The issue is that if you use `unpack xxx.ab -` then `System.out` is being used (https://github.com/nelenkov/android-backup-extractor/blob/master/src/org/nick/abe/AndroidBackup.java#L310) but the progress is using `System.out` as well (https://github.com/nelenkov/android-backup-extractor/blob/master/src/org/nick/abe/AndroidBackup.java#L215) which seems like a mistake since line 209 and 219 use `System.err` instead.

As a result using `abe xxx.ab - | tar tv` is not working for example

**Comment by nelenkov** [comment]
If are sure changing those lines fixes the problem, please submit a pull request.

**Comment by ypujante** [comment]
There is only 1 line to change to fix:

```
diff --git a/src/org/nick/abe/AndroidBackup.java b/src/org/nick/abe/AndroidBackup.java
index dc333ee..651f606 100644
--- a/src/org/nick/abe/AndroidBackup.java
+++ b/src/org/nick/abe/AndroidBackup.java
@@ -212,7 +212,7 @@ public class AndroidBackup {
                     bytesRead = inf == null ? totalRead : inf.getBytesRead();
                     currentPercent = Math.round(bytesRead / fileSize * 100);
                     if (currentPercent != percentDone) {
-                        System.out.print(String.format("%.0f%% ", currentPercent));
+                        System.err.print(String.format("%.0f%% ", currentPercent));
                         percentDone = currentPercent;
                     }
                 }
```

and for the record this is how I test that my `adb` backups are not corrupted:

```
abe unpack backup-2017-03-01.ab - 2> /dev/null | tar -tv
```

**Comment by nelenkov** [comment]
Thanks, but please send a pull request so I can merge with one click :)

**Comment by hilbix** [comment]
See PR #65

**Comment by nelenkov** [comment]
fixed with #65
